roles/openshift_logging/README.md

```diff
@@ -160,3 +161,18 @@ Elasticsearch OPS too, if using an OPS cluster:
   need to set this
 - `openshift_logging_mux_buffer_queue_limit`: Default `[1024]` - Buffer queue limit for Mux.
 - `openshift_logging_mux_buffer_size_limit`: Default `[1m]` - Buffer chunk limit for Mux.
+- `openshift_logging_mux_file_buffer_limit`: Default `[2Gi]` per output - Mux will
+  set the value to the file buffer limit.
```

Comment:
The value of openshift_logging_mux_file_buffer_limit is set to the environment variable FILE_BUFFER_LIMIT in "templates/mux.j2".

FILE_BUFFER_LIMIT is converted to bytes in run.sh like this:
`TOTAL_LIMIT=$(echo ${FILE_BUFFER_LIMIT:-2Gi} | sed -e "s/[Kk]/*1024/g;s/[Mm]/*1024*1024/g;s/[Gg]/*1024*1024*1024/g;s/i//g" | bc) || :`
Then, multiplied by $NUM_OUTPUTS to get the total size of file buffer.
`TOTAL_LIMIT=$(expr $TOTAL_LIMIT \* $NUM_OUTPUTS) || :`
This calculation is needed to compare the TOTAL_LIMIT with the available volume size DF_LIMIT.  Then, the TOTAL_LIMIT is divided by NUM_OUTPUTS again to get the real buffer size limit per buffer:
```
# TOTAL_BUFFER_SIZE_LIMIT per buffer
TOTAL_BUFFER_SIZE_LIMIT=$(expr $TOTAL_LIMIT / $NUM_OUTPUTS) || :
```
The TOTAL_BUFFER_SIZE_LIMIT is identical to FILE_BUFFER_LIMIT if DF_LIMIT is larger than TOTAL_LIMIT.  But if the volume size is smaller, the TOTAL_BUFFER_SIZE_LIMIT is smaller than the user specified FILE_BUFFER_LIMIT.  In the case, the following WARNING is issued by fluentd/run.sh:
WARNING: Available disk space ($DF_LIMIT bytes) is less than the user specified file buffer limit ($FILE_BUFFER_LIMIT times $NUM_OUTPUTS)."